HealthMonitoring.Integration.PushClient.UnitTests/EndpointNotifierTests.cs

```diff
@@ -34,7 +34,7 @@ public class EndpointNotifierTests
             var minimumChecks = 3;
             var expectedEventTimeline = new List<string>();
             for (var i = 0; i < minimumChecks; ++i)
-                expectedEventTimeline.AddRange(new[] { "updateHealth_start", "delay_start", "update_finish", "update_finish" });
+                expectedEventTimeline.AddRange(new[] { "update_start", "update_finish" });
```

Comment:
If the test on the left side produces indeterministic results (I imagine that Task.Delay(100) in that test is not enough sometimes on appveyor), then maybe this test should be changed to one checking that:
1. block of 4 events starts with update_start and finish with update_finish (so what is here)
2. block of 4 consists of 2 starts and 2 finishes, where ones in the middle can be in any order.